libs/user/store/src/lib/login.service.ts

```diff
@@ -0,0 +1,59 @@
+import { inject, Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { environment } from '@ngbank/environment';
+import { User } from './user';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LoginService {
+  httpHeader = new HttpHeaders().set(
+    'X-Appwrite-Project',
+    environment.projectId
+  );
+  http = inject(HttpClient);
+
+  createAccount(user: User) {
+    return this.http.post<User>(
+      `${environment.apiEndpoint}/account`,
+      {
+        userId: user.userId,
+        email: user.email,
+        password: user.password,
+        name: user.name,
+      },
+      {
+        headers: this.httpHeader,
+      }
+    );
+  }
+
+  emailLogin(userName: string, password: string) {
+    return this.http.post<User>(
```

Comment:
@santoshyadavdev this request returns a Session instead of User response: https://appwrite.io/docs/references/cloud/models/session.